src/components/create-from-builder/create-from-builder.controller.ts

```diff
@@ -106,6 +106,7 @@ export class CreateFromBuilderController implements angular.IController {
       onShow: this.showResults
     };
     this.ctrl.steps = [this.infoStep, this.configStep, this.bindStep, this.reviewStep];
+    this.ctrl.currentStep = "Information";
```

Comment:
Do we need to set this everywhere? I'd think it would only be necessary in the error handler when we change steps back to Configuration.

Reply:
We need to set it so that when the error happens, it navigates correctly, else the currentStep will always be "Configuration".

> Is the "Create" button disabled until the user changes the name after an AlreadyExists error?

Yes